flexmeasures/utils/unit_utils.py

```diff
@@ -20,3 +20,12 @@ def determine_stock_unit(flow_unit: str, time_unit: str = "h"):
         if flow_unit.endswith(f"/{time_unit}")
         else f"{flow_unit}{time_unit}"
     )
+
+
+def is_power_unit(unit: str) -> bool:
+    """For example:
+    >>> is_power_unit("kWh")  # True
```

Comment:
Should be `"kW"`.

Reply:
Perhaps a good place to start using `pint`:
```
def is_power_unit(unit: str) -> bool:
    return (pint.Quantity(unit) / pint.Quantity("W")).dimensionless
```

Reply:
I believe that should not be introduced this quickly: From my examples, it can already lead to these kinds of errors:

> pint.errors.UndefinedUnitError: 'EUR' is not defined in the unit registry

> pint.errors.OffsetUnitCalculusError: Ambiguous operation with offset unit (degree_Celsius, watt).

Reply:
Good point. I have been experimenting with pint today, and there are quite a few issues that would still need streamlining. I'd be happy to take this on in a separate PR.